contrib/hdparm/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "hdparm"
```

Comment:
this does not belong in this PR does it

Reply:
ah... forgot to add it to `depends`. Sorry!

Reply:
done.